solr/solrj/src/java/org/apache/solr/common/cloud/Replica.java

```diff
@@ -415,6 +415,13 @@ public void write(JSONWriter jsonWriter) {
     Map<String, Object> map = new LinkedHashMap<>();
     // this serializes also our declared properties
     _allPropsWriter().toMap(map);
+    boolean isLeader = isLeader();
```

Comment:
Does it make sense to include this in `_allPropsWriter` method? as this method included in other method `writeMap` as well.

Reply:
done @hiteshk25